src/sdk.js

```diff
@@ -33,6 +35,17 @@ class OpenForm {
       pathname = pathname.slice(0, pathname.length - 1);
     }
     this.basePath = pathname;
+
+    // Set up idle timer
+    this.idleTimer = null;
+    this.handleOnAction = this.handleOnAction.bind(this);
+  }
+
+  handleOnAction () {
+    if (this.idleTimer.getRemainingTime() === 0) {
+      // Redirect user back to initial screen
+      window.location.href = this.basePath;
```

Comment:
I think this will need to be updated when https://github.com/open-formulieren/open-forms/issues/447 is completed but for now this seemed like the most appropriate way to "log the user out".